akka-persistence/src/main/scala/akka/persistence/journal/leveldb/LeveldbIdMapping.scala

```diff
@@ -19,19 +19,30 @@ private[persistence] trait LeveldbIdMapping extends Actor { this: LeveldbStore 
 
   private val idOffset = 10
   private var idMap: Map[String, Int] = Map.empty
```

Comment:
I'd be tempted to wrap in atomic-ref instead... Maybe doesn't matter since db is the slowest thing here anyway hm...